tests/functional_tests/ffront_tests/test_execution.py

```diff
@@ -1194,9 +1185,6 @@ def program_domain(a: Field[[IDim, JDim], float64]):
 
 
 def test_domain_input_bounds(fieldview_backend):
-    if fieldview_backend == gtfn_cpu.run_gtfn:
-        pytest.skip("FloorDiv and Power not fully supported in gtfn.")
```

Comment:
To keep this PR simple you could keep "FloorDiv not fully supported in gtfn" if it fails, see the other comment. But most likely it works, because otherwise `//` should give an error in the generated code.